### PR TITLE
Stand up app

### PR DESCRIPTION
**Pull request by SamMarkGoldman**
get ready for initial deployment

### CODE REVIEW

**Comment by SamMarkGoldman** [comment]
#test

File: Apronfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  RACK_ENV:
+  RAILS_ENV:
+  REDIS_PASSWORD:
+  REDIS_URL:
+  ROLLBAR_ACCESS_TOKEN:
+  ROLLBAR_ENV:
+  SATS_REDIS_PASSWORD:
+  SATS_REDIS_URL:
+  SECRET_KEY_BASE:
+  SERVICE_SHORT_CODE: MARS
```
Comment (delner) [inline Apronfile]: This needs to be updated.

File: Apronfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - environment/development.env
+  test:
+    - environment/test.env
+environment:
+  DATABASE_URL:
+  FES_URL:
+  HTTP_AUTH_USERNAME:
+  HTTP_AUTH_PASSWORD:
+  JWT_ALGORITHM:
+  JWT_PRIVATE_KEY_MARS:
```
Comment (delner) [inline Apronfile]: As do these.

**Comment by SamMarkGoldman** [comment]
@delner thanks for the catch.  Updated.

File: Dockerfile [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
-# Add the app files (these get overriden by any mounting)
-ADD . /app
+COPY . /opt/wms-btp/
 
-# Start the web server by default
-CMD gem install foreman && \
-    bundle install && \
-    foreman start -f Procfile
\ No newline at end of file
+RUN touch .env
```
Comment (delner) [inline Dockerfile]: Probably should remove this line: this was for something in MARS thats not needed here.

File: Dockerfile [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
+RUN bundle install --system
 
-# Setup a working directory
-WORKDIR /app
+ARG GIT_COMMIT
+ENV GIT_COMMIT ${GIT_COMMIT}
 
-# Add the app files (these get overriden by any mounting)
-ADD . /app
+COPY . /opt/wms-btp/
```
Comment (delner) [inline Dockerfile]: Non-blocking. Perhaps something we can address in a future PR as we set things up for deployment, but we might want to add a `CMD` directive with a default process (e.g. run Puma or something.)

**Pull request merged**